sources/packages/backend/apps/api/src/services/application/application.service.ts

```diff
@@ -292,6 +306,16 @@ export class ApplicationService extends RecordDataModelService<Application> {
       const applicationRepository =
         transactionalEntityManager.getRepository(Application);
       await applicationRepository.save(newApplication);
+
+      //Check if the application requires E2 restriction check.
```

Comment:
Minor. Please start the comment with space. `// Check if the application requires E2 restriction check.` same in other places.